core/src/ecs/systems/ui/InputSystem.ts

```diff
@@ -190,6 +190,11 @@ export const InputSystem = ClientSystemBuilder({
       for (const keyPress in input.inputMap.press) {
         const keyMouse = bufferDown.contains(keyPress);
         if (keyMouse) {
+
+          // ignore stale inputs
+          if (keyMouse.tick + 1 != world.tick) continue;
```

Comment:
potentially some duplicate logic with earlier change when handling `capslock` keypress for `Minimap`